x/liquidity/keeper/invariants.go

```diff
@@ -94,10 +95,15 @@ func MintingPoolCoinsInvariant(poolCoinTotalSupply, mintPoolCoin, depositCoinA,
 	expectedMintPoolCoinAmtBasedA := depositCoinARatio.MulInt(poolCoinTotalSupply).TruncateInt()
 	expectedMintPoolCoinAmtBasedB := depositCoinBRatio.MulInt(poolCoinTotalSupply).TruncateInt()
 
-	// NewPoolCoinAmount / LastPoolCoinSupply <= AfterRefundedDepositCoinA / LastReserveCoinA
-	// NewPoolCoinAmount / LastPoolCoinSupply <= AfterRefundedDepositCoinA / LastReserveCoinB
-	if depositCoinARatio.LT(poolCoinRatio) || depositCoinBRatio.LT(poolCoinRatio) {
-		panic("invariant check fails due to incorrect ratio of pool coins")
+	// NewPoolCoinAmount / LastPoolCoinSupply == AfterRefundedDepositCoinA / LastReserveCoinA
+	// NewPoolCoinAmount / LastPoolCoinSupply == AfterRefundedDepositCoinA / LastReserveCoinB
+	if depositCoinA.GTE(coinAmountThreshold) && depositCoinB.GTE(coinAmountThreshold) &&
+		lastReserveCoinA.GTE(coinAmountThreshold) && lastReserveCoinB.GTE(coinAmountThreshold) &&
+		mintPoolCoin.GTE(coinAmountThreshold) && poolCoinTotalSupply.GTE(coinAmountThreshold) {
+		if errorRate(depositCoinARatio, poolCoinRatio).GT(errorRateThreshold) ||
+			errorRate(depositCoinBRatio, poolCoinRatio).GT(errorRateThreshold) {
+			panic("invariant check fails due to incorrect ratio of pool coins")
```

Comment:
@hallazzang Could you add a testcase for this panic for coverage?

Reply:
of if there are test codes for the case already, but not detected on codecov, we can merge https://github.com/tendermint/liquidity/pull/451